rust-toolchain.toml

```diff
@@ -1,3 +1,4 @@
+# TODO: Update Tera in CI to v0.5.0 once Rust version is > 1.83.0
```

Comment:
Is there an issue for this?  If so can you please link to it here and above?

Reply:
There is not, but that's why I put it here. Right above where the update would be edited. However, as this is just doing release note management, it is fine to keep using until it breaks.